pantheon-bundle/frontend/src/app/search.tsx

```diff
@@ -122,10 +118,10 @@ class Search extends Component<IAppState, ISearchState> {
                   <DataListItemCells
                     dataListCells={[
                       <DataListCell width={2} key="title">
-                        <button onClick={this.sortByName} className="sp-prop" id="span-name" aria-label="sort column by name">Name</button>
+                        <span className="sp-prop-nosort" id="span-name" aria-label="column name">Name</span>
                       </DataListCell>,
                       <DataListCell width={2} key="description">
-                        <button onClick={this.sortByDescription} className="sp-prop" id="span-name" aria-label="sort column by description">Description</button>
+                        <span className="sp-prop-nosort" id="span-name" aria-label="column description">Description</span>
```

Comment:
The span tag id looks like a typo. It's a duplicate of the Name cell above. Should it be id="span-description" ?

Reply:
yep overlooked those ids, they have been updated in the latest commit too.